### ISSUE TITLE
accept relative paths for pact files

### ISSUE THREAD

**Issue by travi** [pact-foundation/pact-node#80]
as mentioned in #79, i was surprised to find that the cli did not accept paths that were relative to the project root in npm scripts. while an absolute path does work, a relative path normally works with cli tools as well.

i don't necessarily think its too limiting to only accept absolute paths, but it should at least be made more clear in the documentation that the limitation exists.

this is not a blocker for me, so this issue is just to give visibility to the confusion i ran into.

**Comment by mboudreau** [comment]
@travi yep, I tend to agree.  The issue is that the standalone binary doesn't understand relative path since it doesn't have the context (the current working directory) from your node project, hence the need for absolute paths.  However, it should be easy enough to add that functionality into the node wrapper :)

I'll try to get something out today.

**Comment by mboudreau** [comment]
@travi got a PR going to fix this issue: https://github.com/pact-foundation/pact-node/pull/85
